lib/whitehall_importer.rb

```diff
@@ -82,10 +82,14 @@ def self.sync(whitehall_import)
   end
 
   def self.create_timeline_entry(edition)
```

Comment:
Sorry I know you didn't create this, just wondering if you could scratch an itch of mine here.

I think this method is in the wrong place here - It doesn't really seem a concern of the overall WhitehallImporter module and we'd want to abort an import if it failed (as unlikely as that may be) so I think it belongs in the WhitehallImporter::Import class within the transaction.

Any chance you could sort that as part of this 🙏 ?

Reply:
Have moved!